e2e/test/scenarios/native/reproductions/16584-run-selected-text-does-not-preserve-parameter-values.cy.spec.ts

```diff
@@ -8,11 +8,18 @@ describe("issue 16584", () => {
   });
 
   it("should pass parameters when running with 'Run select text' (metabase#16584)", () => {
+    // The bug described in is #16584 can be further simplified:
+    // - the issue persists even when selecting the *entire* query
+    // - the issue is unrelated to using a date filter, using a text filter works too
+    // - the issue is unrelated to whether or not the parameter is required or if default value is set
     openNativeEditor()
-      .type("SELECT * FROM ACCOUNTS WHERE COUNTRY = {{ country ")
+      .type("SELECT * FROM ACCOUNTS WHERE COUNTRY = {{ country }} ", {
+        parseSpecialCharSequences: false,
+        delay: 0,
+      })
       .type("{selectAll}");
 
-    cy.findByPlaceholderText("Country").type("NL");
+    cy.findByPlaceholderText("Country").type("NL", { delay: 0 });
```

Comment:
FWIW, this is perfectly fine to leave without a delay. It's short enough.
You don't have to change it again. Just saying.